lib/Dialect/AIEX/Transforms/AIEDmaToNpu.cpp

```diff
@@ -364,18 +336,23 @@ struct DmaToNpuPattern : OpConversionPattern<NpuDmaMemcpyNdOp> {
     if (!isMM2S)
       issue_token = BoolAttr::get(ctx, true);
 
-    rewriter.create<NpuWriteBdExShimTileOp>(
-        op->getLoc(), column, column_num, ddr_id, bd_id, buffer_length,
-        buffer_offset, enable_packet, out_of_order_id, packet_id, packet_type,
-        d0_size, d0_stride, d1_size, d1_stride, d2_stride, iteration_current,
-        iteration_size, iteration_stride, next_bd, use_next_bd, valid_bd,
+    rewriter.create<NpuWriteBdOp>(
```

Comment:
This would need to be extended further for MemTile with another DMA dimension

Reply:
> This would need to be extended further for MemTile with another DMA dimension

Thanks for caching this. I left actual support and tests for memtile as future work.